### ISSUE TITLE
Add logo and header image to template

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-intranet-theme#6]
Hi Eric,
is it possible to add a spot in the upper left corner to add our logo instead of it saying "Jackson College intranet"? Also, can we add a spot that we can add a header/feature image that would span across all pages? I'm thinking the logo/header area doesn't have to be any taller than 175 px. See the image below (I've already done some styling to the left navigation) Thanks!

![Capture](https://user-images.githubusercontent.com/15264315/55336078-57ee1c80-546a-11e9-8513-993d32cbba58.JPG)

**Comment by banksashleym** [comment]
I hate to be a bother, but is there any way this could be the next issue we tackle? I think once this one is done I can start sending this out as the main template will be completed. Thanks!

**Comment by d4mation** [comment]
In your example above, it looks like the "Top Bar" is going away entirely. Is that correct? 

Since this has a few implications, I've put this on Staging for review.

The following notes are based on how it was before:

1. The "Top Bar" extends all the way from left to right. This means that the Side Nav as actually placed below it. I changed the Side Nav to Yellow in the screenshot below as an example. 

![image](https://user-images.githubusercontent.com/7770631/56046741-ee93c680-5d11-11e9-95e4-23f03f0178bd.png)

2. On Mobile, the Menu Button is placed within that Top Bar. The Side Nav is a separate element on Mobile and Desktop although they do look similar. 

To address this, I am hiding the Top Bar on Desktop (When the Side Nav would be displayed) and on Mobile/Smaller Tablets am showing the Top Bar instead of the Side Nav. 

Staging is not up-to-date with Live content wise, but since Live has a lot of caching in play I wanted to get back to a Staging workflow with this for testing :+1:

You can find the field to upload a Logo under Appearance -> Customize -> Site Identity

https://extranet-jccmi-edu.jacksoncollege.staging.wpengine.com/

**Comment by banksashleym** [comment]
This is perfect and exactly what I envisioned. Thank you!

**Comment by d4mation** [comment]
Just pushed this change to Live :+1:

You'll need to upload the logo, but it is otherwise good-to-go.

**Comment by banksashleym** [comment]
thanks!
